### DISCUSSION TITLE
Layout is called with wrong height value on Mac OS after disabling fullscreen with green button

### DISCUSSION THREAD

**Discussion by IlyaFloppy** [hajimehoshi/ebiten#1883] [Q&A]
I am running a game in fullscreen using `ebiten.SetFullscreen(true)`. Layout method is called with width=1919 and height=1200. Then I use green window button to exit fullscreen and `Layout` method keeps getting called with the same width and height. The game looks squeezed vertically. After I resize the window dragging the corner it looks normal and width and height change to 1920 and 1073 respectively.

```go
func (g *Game) Layout(outsideWidth, outsideHeight int) (int, int) {
	fmt.Println("LAYOUT", outsideWidth, outsideHeight)
	return outsideWidth, outsideHeight
}
```

```
// App starts
LAYOUT 1920 1073
// SetFullscreen is called
[1535] LAYOUT 1919 1200
// Green button is used to disable fullscreen
[2643] LAYOUT 1919 1200
// Start changing window size
LAYOUT 1907 1073
LAYOUT 1906 1073
LAYOUT 1905 1073
LAYOUT 1907 1073
LAYOUT 1908 1073
LAYOUT 1909 1073
[14105] LAYOUT 1920 1073
```

**Comment by hajimehoshi** [comment]
7be00d79e864f8475ae96ec7ece2fc273459bdad

The a-little-smaller width for fullscreen is intentional to avoid a mysterious FPS drop.
